### PR TITLE
Testdnd

### PR DESCRIPTION
**Pull request by Melih4irat**

### CODE REVIEW

File: components/dragtes/KanBan.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+const Container = styled.div`
+  width: 100%;
+`;
+const ListContainer = styled.div`
+  height: 160px;
+  width: 100%;
+
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
```
Comment (simon-schuster44) [inline components/dragtes/KanBan.js:123]: Please use relative units here.

File: components/dragtes/KanBan.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+`;
+const TaskList = styled.div`
+  display: flex;
+  width: auto;
+  height: auto;
+  position: relative;
+  border-radius: 5px;
+  padding-bottom: 10px;
```
Comment (simon-schuster44) [inline components/dragtes/KanBan.js:131]: Please use relative units here.

File: components/dragtes/KanBan.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  width: 100%;
+  min-height: 80vh;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Title = styled.span`
+  color: #fff;
+
+  padding: 2px 10px;
```
Comment (simon-schuster44) [inline components/dragtes/KanBan.js:148]: Please use relative units here.

**Comment by simon-schuster44** [comment]
Great! I've made some comments for you...

File: components/dragtes/KanBan.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+export default Kanban;
+
+const Container = styled.div`
+  width: 100%;
+`;
+const ListContainer = styled.div`
+  height: 160px;
+  width: 100%;
+
+  display: flex;
```
Comment (B4CKSL4SH-44) [inline components/dragtes/KanBan.js:121]: Great! Please take a look at my comments.

File: components/dragtes/KanBan.js (hunk 1 above)
Comment (B4CKSL4SH-44) [inline components/dragtes/KanBan.js:123]: Please use relative units here.

File: components/dragtes/KanBan.js (hunk 2 above)
Comment (B4CKSL4SH-44) [inline components/dragtes/KanBan.js:131]: Please use relative units here.

File: components/dragtes/KanBan.js [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Title = styled.span`
+  color: #fff;
+
+  padding: 2px 10px;
+
+  align-self: flex-start;
```
Comment (B4CKSL4SH-44) [inline components/dragtes/KanBan.js:150]: Please use relative units here.

File: components/dragtes/TaskCard.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  padding: 10px 10px;
+  height: 130px;
+  border-radius: 5px;
+  width: 200px;
+  background: rgba(255, 255, 255, 0.2);
+  padding-bottom: 30px;
+  position: relative;
```
Comment (B4CKSL4SH-44) [inline components/dragtes/TaskCard.js:61]: Please use relative units here.

**Pull request merged**